From 672a4748147bfa380e468dfb7f223d4b3278c785 Mon Sep 17 00:00:00 2001
From: Alexander Duyck <alexander.h.duyck@intel.com>
Date: Fri, 11 Jan 2013 10:38:42 -0800
Subject: [PATCH 775/858] net: Export __netdev_pick_tx so that it can be used
 in modules

When testing with FCoE enabled we discovered that I had not exported
__netdev_pick_tx.  As a result ixgbe doesn't build with the RFC patches
applied because ixgbe_select_queue was calling the function.  This change
corrects that build issue by correctly exporting __netdev_pick_tx so it
can be used by modules.

Signed-off-by: Alexander Duyck <alexander.h.duyck@intel.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
(cherry picked from commit 87696f9234cb681ded2697b0f0fd01bb15a2aff2)

Signed-off-by: Arnaldo Carvalho de Melo <acme@redhat.com>
---
 net/core/dev.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/net/core/dev.c b/net/core/dev.c
index 82f5d40..74fdfa2 100644
--- a/net/core/dev.c
+++ b/net/core/dev.c
@@ -2752,6 +2752,7 @@ u16 __netdev_pick_tx(struct net_device *dev, struct sk_buff *skb)
 
 	return queue_index;
 }
+EXPORT_SYMBOL(__netdev_pick_tx);
 
 struct netdev_queue *netdev_pick_tx(struct net_device *dev,
 				    struct sk_buff *skb)
-- 
1.8.3.1

